python/mlcroissant/mlcroissant/_src/core/constants.py

```diff
@@ -73,26 +73,24 @@ def ML_COMMONS(ctx) -> rdflib.Namespace:
 # Attributes
 ML_COMMONS_RAI_DATA_COLLECTION = RAI.dataCollection
 ML_COMMONS_RAI_DATA_COLLECTION_TYPE = RAI.dataCollectionType
-ML_COMMONS_RAI_DATA_COLLECTION_TYPE_OTHERS = RAI.dataCollectionTypeOthers
-ML_COMMONS_RAI_DATA_COLLECTION_MISSING = RAI.dataCollectionMissing
-ML_COMMONS_RAI_DATA_COLLECTION_RAW = RAI.dataCollectionRaw
-ML_COMMONS_RAI_DATA_COLLECTION_TIMEFRAME_START = RAI.dataCollectionTimeFrameStart
-ML_COMMONS_RAI_DATA_COLLECTION_TIMEFRAME_END = RAI.dataCollectionTimeFrameEnd
+ML_COMMONS_RAI_DATA_COLLECTION_MISSING = RAI.dataCollectionMissingData
+ML_COMMONS_RAI_DATA_COLLECTION_RAW = RAI.dataCollectionRawData
+ML_COMMONS_RAI_DATA_COLLECTION_TIMEFRAME = RAI.dataCollectionTimeFrame
 ML_COMMONS_RAI_DATA_PREPROCESSING_IMPUTATION = RAI.dataPreprocessingImputation
-ML_COMMONS_RAI_DATA_PREPROCESSING_PROTOCOL = RAI.dataPeprocessingProtocol
+ML_COMMONS_RAI_DATA_PREPROCESSING_PROTOCOL = RAI.dataPreprocessingProtocol
 ML_COMMONS_RAI_DATA_PREPROCESSING_MANIPULATION = RAI.dataPreprocessingManipulation
 ML_COMMONS_RAI_DATA_ANNOTATION_PROTOCOL = RAI.dataAnnotationProtocol
 ML_COMMONS_RAI_DATA_ANNOTATION_PLATFORM = RAI.dataAnnotationPlatform
 ML_COMMONS_RAI_DATA_ANNOTATION_ANALYSIS = RAI.dataAnnotationAnalysis
-ML_COMMONS_RAI_DATA_ANNOTATION_PER_ITEM = RAI.dataAnnotationPerItem
-ML_COMMONS_RAI_DATA_ANNOTATION_DEMOGRAPHICS = RAI.dataAnnotationDemographics
-ML_COMMONS_RAI_DATA_ANNOTATION_TOOLS = RAI.dataAnnotationTools
+ML_COMMONS_RAI_DATA_ANNOTATION_PER_ITEM = RAI.annotationsPerItem
+ML_COMMONS_RAI_DATA_ANNOTATION_DEMOGRAPHICS = RAI.annotatorDemographics
+ML_COMMONS_RAI_DATA_ANNOTATION_TOOLS = RAI.machineAnnotationTools
 ML_COMMONS_RAI_DATA_USE_CASES = RAI.dataUseCases
 ML_COMMONS_RAI_DATA_BIASES = RAI.dataBiases
-ML_COMMONS_RAI_DATA_LIMITATION = RAI.dataLimitation
+ML_COMMONS_RAI_DATA_LIMITATION = RAI.dataLimitations
 ML_COMMONS_RAI_DATA_SOCIAL_IMPACT = RAI.dataSocialImpact
-ML_COMMONS_RAI_DATA_SENSITIVE = RAI.dataSensitive
-ML_COMMONS_RAI_DATA_MAINTENANCE = RAI.dataMaintenance
+ML_COMMONS_RAI_DATA_SENSITIVE = RAI.personalSensitiveInformation
```

Comment:
Two possibilities:

- Either the name of the constant should reflect the name of the property. E.g., here `ML_COMMONS_RAI_PERSONAL_SENSITIVE_INFORMATION`.
- Or you shouldn't use any constant at all and directly use `url=RAI.personalSensitiveInformation` in metadata.py.